Trappist/src/Promact.Trappist.Test/Tests/TestsRepositoryTest.cs

```diff
@@ -120,5 +160,17 @@ private DomainModel.Models.Test.Test CreateTest(string testName)
             };
             return test;
         }
+
+        private DomainModel.Models.TestConduct.TestAttendees TestAttendee()
+        {
+            var test = CreateTest("Quantitative");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused 'test' local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1481)